src/storage/s3v2OS.py

```diff
@@ -343,6 +415,7 @@ def delete_object(self,containername,prefix,objectname):
 
                 return 0
             # TODO: not working for some reason
```

Comment:
Is this not working or is it a stale comment?

Reply:
Delete in S3v2 is not working. It's something that couldn't be solved using the API.